src/main/java/com/github/manolo8/darkbot/core/itf/ExtraMenuProvider.java

```diff
@@ -23,8 +24,14 @@ default JMenuItem create(String key, ActionListener listener) {
         return item;
     }
 
+    default JMenu createMenu(String key, List<JComponent> components) {
+        JMenu menu = new JMenu(key);
+        for (JComponent component : components) menu.add(component);
+        return menu;
+    }
+
     default JComponent createSeparator(String key) {
-        return new JTitledPopupMenuSeparator(I18n.getOrDefault("gui.hamburger_button." + key, key));
+        return new JTitledPopupMenuSeparator(I18n.getOrDefault("gui.hamburger_separator." + key, key));
```

Comment:
```suggestion
        return new JTitledPopupMenuSeparator(I18n.getOrDefault("gui.hamburger_button." + key, key));
```
This translation key is for all entries in the hamburger button menu, not just separators, and we'd rather not have to change all used translation keys prior to this